src/VodafoneMessage.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace NotificationChannels\Vodafone;
+
+class VodafoneMessage
+{
+    /**
+     * @var string
+     */
+    public $content;
+
+    /**
+     * @var array
+     */
+    public $from;
+
+    /**
+     * Create a new message instance.
+     *
+     * @param  string  $content
+     * @return void
+     */
+    public function __construct($content = '')
+    {
+        $this->content = $content;
+    }
+
+    /**
+     * Set the notification content.
+     *
+     * @param  string $value
+     * @return $this
+     */
+    public function content($value)
+    {
+        $this->content = $value;
+
+        return $this;
+    }
+
+    /**
+     * Set the notification sender.
+     *
+     * @param  string $value
+     * @return $this
+     */
+    public function from($value)
```

Comment:
Is from required? If so, might be worth adding a check for that (if not, ignore)

Reply:
'from' can be left blank, or not included in the request at all and Vodafone will just use a standard number. Only if an additional number or Alphacode is configured in their system would you then include that value in the 'from' field.

Reply:
Sweet, no worries :)